modules/howtos/pages/working-with-collections.adoc

```diff
@@ -1,12 +1,15 @@
-= Working with the  Collections Developer Preview
-:nav-title: Collections DP
+= Working with the  Collections
```

Comment:
Extra space after `the`

Reply:
or you could say "extra `the`" 😂 , will delete

Reply:
yeah just re-read that in my head haha - you got it 👍